### PR TITLE
fix salt service start

### PR DESCRIPTION
**Pull request by vkhatri**
PR to make sure `salt-minion` and `salt-master` service always get started.

### CODE REVIEW

File: recipes/master.rb [hunk 1]
```diff
@@ -84,6 +80,16 @@
   end
 end
 
+ruby_block 'delay salt-master service start' do
+  block do
+  end
+  notifies :start, 'service[salt-master]'
```
Comment (shortdudey123) [inline recipes/master.rb]: any reason you don't just add `:start` onto the service block?

Comment (vkhatri) [inline recipes/master.rb]: @shortdudey123 it is in the favor of LWRP resource that I am working on, e.g. `external_auth` and it works slightly better by delaying the service start.

Comment (vkhatri) [inline recipes/master.rb]: and there was some issue with kitchen test which got resolved by delaying the service start.

I do not remember what it was, but I can check it again if required.

Comment (shortdudey123) [inline recipes/master.rb]: > some issue with kitchen test which got resolved

Can you add more info here about that? (for future reference and full context on the change)

**Comment by vkhatri** [comment]
Closing, anyhow requires re base.

**Pull request closed without merging**